### ISSUE TITLE
Cannot read property 'to' of undefined Grafana 4.5.0

### ISSUE THREAD

**Issue by innovia** [NatelEnergy/grafana-discrete-panel#24]
Hi

I get the following error on immediately after adding the panel

```javascript
TypeError: Cannot read property 'to' of undefined
    at DiscretePanelCtrl.issueQueries (https://grafana.lightricks.com/public/plugins/natel-discrete-panel/module.js?bust=1505394229707:425:34)
    at i (https://grafana.lightricks.com/public/app/boot.4aeac494.js:61:3972)
    at https://grafana.lightricks.com/public/app/boot.4aeac494.js:61:4394
    at o.$eval (https://grafana.lightricks.com/public/app/boot.4aeac494.js:61:11549)
    at o.$digest (https://grafana.lightricks.com/public/app/boot.4aeac494.js:61:10022)
    at o.$apply (https://grafana.lightricks.com/public/app/boot.4aeac494.js:61:11832)
    at https://grafana.lightricks.com/public/app/boot.4aeac494.js:61:7882
    at f (https://grafana.lightricks.com/public/app/boot.4aeac494.js:59:22132)
    at https://grafana.lightricks.com/public/app/boot.4aeac494.js:59:23593

```

**Comment by chorca** [comment]
Having a similar issue, only mine is reading "from". Using Grafana 4.5.1:

`Panel data error: TypeError: Cannot read property 'from' of undefined
    at a.getTimeFilter (boot.de7c040d.js:17)
    at a.query (boot.de7c040d.js:17)
    at DiscretePanelCtrl.issueQueries (module.js:390)
    at i (boot.de7c040d.js:61)
    at boot.de7c040d.js:61
    at o.$eval (boot.de7c040d.js:61)
    at o.$digest (boot.de7c040d.js:61)
    at o.$apply (boot.de7c040d.js:61)
    at boot.de7c040d.js:61
    at f (boot.de7c040d.js:59)`

**Comment by alin-amana** [comment]
As a workaround, you can clear the "Expand 'from' query" field in the panel's Options tab.

The issue comes from the fact that discrete panel depends on the internals of Grafana and they have changed. I've made a local modification which should make it work with both old and new versions of Grafana, I'll submit a PR and link it.

**Comment by ryantxu** [comment]
I just committed a fix and it will be released in v0.0.6  (soon)
